tests/tracer/utils_botocore/test_span_pointers.py

```diff
@@ -211,8 +234,8 @@ def test_pointers(self, pointers_case: PointersCase) -> None:
 
                 (args, kwargs) = mock_logger.call_args
                 assert not kwargs
-                fmt, other_args = args
+                fmt, *other_args = args
```

Comment:
Should line 210 be  `mock_logger.assert_called_once()` ?

Reply:
Also, if you know, how did this line not fail before adding the `*` in front of other?

Reply:
hmm... it should be "assert" indeed.... wonder how that wasn't failing...

and yes, before there was a single arg and now there's multiple args =/

Reply:
oh, because it's a mock object that will happily mock anything. i had tried to use a log output catcher instead of checking the logger calls but that was too flaky, unfortunately.

Reply:
fixed this!